index.d.ts

```diff
@@ -0,0 +1,77 @@
+import { EventEmitter } from 'events';
+import {
+	PermissionResolvable,
+	Snowflake,
+	User,
+	Guild,
+	TextChannel,
+	GuildMember,
+	Message,
+	DiscordAPIError,
+	RichEmbed
```

Comment:
```suggestion
	RichEmbed,
	Role
```
Role is used (line 70) so we need to import it from Discord.js